### ISSUE TITLE
More information in detail view.

### ISSUE THREAD

**Issue by TamasB14** [studio-minus/ppg-bugs-and-requests#163]
**Feature description**
Adding much more icons to the detail view instead of just Health bar, Exploision Radious and an X.
For alive things there should be:
- Dead [Skull icon] (For example the ragdoll has been shot in the head)
- Uncoscious [Zzz icon] (Could be kept above bodypart health.)
- No blood [ X icon] (For example the ragdoll bled out)

For not alive things there should be:
- Temperature 

**Feature purpose**
Could help me identify most of the time if that my ragdoll is dead for like getting shot in the head, or it was just bleeding out.
And the temperature could help me identify if the platform is cool enough to place my humans at, 'cause sometimes it's not visible that the metal bar is hot, and I place my ragdolls on it and they get lighten up.

**Additional context**
I get it there's a temp sensor in the game, but sometimes there are places I can't reach with it.

**Comment by JohnVcav** [comment]
totally agree with the temperature part, I feel like you have to get too out of the way to check the temp of non-living things.
as for the icons, I don't believe that it's too necessary since you can just inspect the person.
although it would be really nice to see HP/Durability in number values somehow.

**Comment by TamasB14** [comment]
> totally agree with the temperature part, I feel like you have to get too out of the way to check the temp of non-living things.
> as for the icons, I don't believe that it's too necessary since you can just inspect the person.
> although it would be really nice to see HP/Durability in number values somehow.

I would like to see the temp and item durability atleast.

**Comment by Cboyjet4** [comment]
I'm not really a huge fan of detail view, inspect basically already serves the same purpose (for humans at least), but much more in-depth, I wouldn't like convoluting it more.

**Comment by mestiez** [comment]
> I'm not really a huge fan of detail view, inspect basically already serves the same purpose (for humans at least), but much more in-depth, I wouldn't like convoluting it more.

I agree. Temperature gizmos would crowd the whole thing way too much. Perhaps a completely seperate temperature view would be a better idea but I'd like to keep the UI very simple. I am not sure yet.
